docs/source/getting_started/index.md

```diff
@@ -1,87 +1,175 @@
 # Quick Start
 
+In this guide, we'll walk through how you can use the Llama Stack (server and client SDK) to test a simple agent.
+A Llama Stack agent is a simple integrated system that can perform tasks by combining a Llama model for reasoning with
+tools (e.g., RAG, web search, code execution, etc.) for taking actions.
+In Llama Stack, we provide a server exposing multiple APIs. These APIs are backed by implementations from different providers.
 
 Llama Stack is a stateful service with REST APIs to support seamless transition of AI applications across different environments. The server can be run in a variety of ways, including as a standalone binary, Docker container, or hosted service. You can build and test using a local server first and deploy to a hosted endpoint for production.
 
-In this guide, we'll walk through how to build a RAG agent locally using Llama Stack with [Ollama](https://ollama.com/) to run inference on a Llama Model.
+In this guide, we'll walk through how to build a RAG agent locally using Llama Stack with [Ollama](https://ollama.com/)
+as the inference [provider](../providers/index.md#inference) for a Llama Model.
 
+## Step 1: Installation and Setup
 
-### 1. Download a Llama model with Ollama
+### i. Install and Setup Ollama for Inference
 
+Install Ollama by following the instructions on the [Ollama website](https://ollama.com/download).
+
+Then download a Llama model with Ollama
 ```bash
 ollama pull llama3.2:3b-instruct-fp16
```

Comment:
change model to llama3.2:3b - its smaller and faster

Reply:
done